src/components/CheckboxGroup/checkbox.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Checkbox(props) {
+    const { id, label, disabled, isSelected, onChange } = props;
+    return (
+        <div className="rainbow-checkbox" role="presentation" onClick={() => onChange(id, !isSelected, disabled)}>
+            <input
+                id={id}
```

Comment:
this id is not being passed from parent. This must be a generated unique id stored in constructor.